diff -up mold-2.0.0/third-party/mimalloc/src/prim/unix/prim.c.omv~ mold-2.0.0/third-party/mimalloc/src/prim/unix/prim.c
--- mold-2.0.0/third-party/mimalloc/src/prim/unix/prim.c.omv~	2023-07-26 21:17:41.865210924 +0100
+++ mold-2.0.0/third-party/mimalloc/src/prim/unix/prim.c	2023-07-26 21:17:57.465225986 +0100
@@ -27,6 +27,7 @@ terms of the MIT license. A copy of the
 
 #include <sys/mman.h>  // mmap
 #include <unistd.h>    // sysconf
+#include <stdlib.h>    // getenv
 
 #if defined(__linux__)
   #include <features.h>
