diff -up mold-2.3.0/third-party/blake3/c/CMakeLists.txt.omv~ mold-2.3.0/third-party/blake3/c/CMakeLists.txt
--- mold-2.3.0/third-party/blake3/c/CMakeLists.txt.omv~	2023-10-19 11:46:15.761334869 +0200
+++ mold-2.3.0/third-party/blake3/c/CMakeLists.txt	2023-10-19 11:46:35.979489170 +0200
@@ -31,7 +31,7 @@ set(BLAKE3_X86_NAMES i686 x86 X86)
 set(BLAKE3_ARMv8_NAMES aarch64 AArch64 arm64 ARM64 armv8 armv8a)
 
 # library target
-add_library(blake3
+add_library(blake3 STATIC
   blake3.c
   blake3_dispatch.c
   blake3_portable.c
